gulpfile.js

```diff
@@ -15,17 +15,18 @@ const gutil = require('gulp-util');
 
 const paths = {
     vendorScripts: [
-        'res/components/jquery/dist/jquery.js',
-        'res/components/bootstrap/dist/js/bootstrap.js',
-        'res/components/fitvids/jquery.fitvids.js',
-        'res/components/picturefill/dist/picturefill.js'
+        'node_modules/jquery/dist/jquery.js',
+        'node_modules/bootstrap/dist/js/bootstrap.js',
+        'node_modules/fitvids.1.1.0/jquery.fitvids.js',
+        'node_modules/picturefill/dist/picturefill.js',
+        'node_modules/respond.js/dest/respond.src.js'
```

Comment:
This is the incorrect place to put the link to respond.js.

See my earlier note for the correct location in the gulpfile.js

![2017-06-05 at 3 40 01 pm](https://cloud.githubusercontent.com/assets/917632/26799888/44a7cdb8-4a05-11e7-8a61-8a334bc4971b.png)